src/content/docs/en/guides/internationalization.mdx

```diff
@@ -289,8 +305,32 @@ Use this like [`getRelativeLocaleUrl`](#getrelativelocaleurl) to return a list o
 
 `getAbsoluteLocaleUrlList(locale: string, options?: GetLocaleOptions): string[]`
 
-
 Use this like [`getAbsoluteLocaleUrl`](#getabsolutelocaleurl) to return a list of absolute paths for all the locales.
 
+### `getPathByLocale(locale: string): string`
+
+A function that returns the `path` associated to one or more `codes` when [custom locale paths](#custom-locale-paths) are configured.
+
+### Generating URLs from custom paths
+
+When a `locale` is a custom path, use the `path` as the value for `locale`.
```

Comment:
``````suggestion
When a `locale` is a custom path, use the `path` as the value for `locale`.

For example, use `french` in the i18n helper functions for the `locales` configuration below:

```js
    locales: ["es", "en", {
        path: "french",
        codes: ["fr", "fr-BR", "fr-CA"]
    }],
```
``````